api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -143,8 +143,9 @@ message ExternalProcessor {
   // a message on the stream that requires a response, it will reset this timer,
   // and will stop processing and return an error (subject to the processing mode)
   // if the timer expires before a matching response is received. There is no
-  // timeout when the filter is running in asynchronous mode. The
-  // ``message_timeout`` range is >= 0s and <= 3600s. Default is 200 milliseconds.
+  // timeout when the filter is running in asynchronous mode. Zero is a valid
```

Comment:
This change is to clean up some description in this proto message.   Same as below. 
It is not related to the headers_to_forward API.

Reply:
Please split these into separate PRs in future. It's trivial enough to merge in for now though.

Reply:
Sure, this change is now in https://github.com/envoyproxy/envoy/pull/27426.